### ISSUE TITLE
Invoke JS code from PS scripts

### ISSUE THREAD

**Issue by alan-null** [SitecorePowerShell/Console#679]
As a SPE user I want to be able to invoke some JS code before/during/after script execution.
This could be really useful in Content Editor for example.

Here is my use-case.
Let's say I've enabled `Insert Item` integration point for `Content Editor` and I've got one script there.
This script just creates an new item below current item.
Normally when I run this script and everything go fine my content tree in CE remains unchanged, thus I need to refresh it manually.

Here is my POC how I would like my feature to work.
![autorefresh_tree](https://cloud.githubusercontent.com/assets/6848691/17777650/9b3b8752-6561-11e6-88c4-be25370d9cc7.gif)

As you can see right after closed **Runner** dialog my tree was refreshed.

All I added is this line:

``` csharp
SheerResponse.Eval($"top.window.scForm.postRequest('', '', '', 'Tree_Refresh(\"' + '{CurrentItem.ID}' + '\")');");
```

before this line:
https://github.com/SitecorePowerShell/Console/blob/4.0/Cognifide.PowerShell/Client/Applications/PowerShellRunner.cs#L322

What do you think?

**Comment by michaellwest** [comment]
Cool. I actually built this when trying to copy data to the clipboard but
that didn't work out too well. Perhaps you implementation is better.

**Comment by michaellwest** [comment]
Perhaps we should add a parameter to Close-Window that let's you specify a
refresh?

**Comment by alan-null** [comment]
@michaellwest 
I am not sure if this will be enough. We tested it for a few minutes with @AdamNaj. For some cases I need also information about window/html document where I invoked the PS script.
This might occur when I run script on the desktop where I've got several Content Editor windows.
There is no way to figure out which CE window is mine.

My code works for now only when you open CE directly, not as a part of desktop
`http://domain/sitecore/shell/Applications/Content%20Editor.aspx?sc_bw=1`.
Notice the `top.window` usage, it has different context in desktop.
